website/content/docs/platform/k8s/helm/examples/standalone-tls.mdx

```diff
@@ -67,86 +74,99 @@ TMPDIR=/tmp
-
    2. Send the CSR to Kubernetes.
 
-      ```bash
-      kubectl create -f ${TMPDIR}/csr.yaml
+      ```shell-session
+      $ kubectl create -f ${TMPDIR}/csr.yaml
+      certificatesigningrequest.certificates.k8s.io/vault-csr created
       ```
 
       -> If this process is automated, you may need to wait to ensure the CSR has been received and stored:
       `kubectl get csr ${CSR_NAME}`
 
    3. Approve the CSR in Kubernetes.
 
-      ```bash
-      kubectl certificate approve ${CSR_NAME}
+      ```shell-session
+      $ kubectl certificate approve ${CSR_NAME}
+      certificatesigningrequest.certificates.k8s.io/vault-csr approved
+      ```
+
+   4. Verify that the certificate was approved and issued.
+      ```shell-session
+      $ kubectl get csr ${CSR_NAME}
+      NAME        AGE     SIGNERNAME                                    REQUESTOR                        CONDITION
+      vault-csr   1m13s   kubernetes.io/kubelet-serving                 kubernetes-admin                 Approved,Issued
       ```
 
 ## 2. Store key, cert, and Kubernetes CA into Kubernetes secrets store
 
 1. Retrieve the certificate.
 
-   ```bash
-   serverCert=$(kubectl get csr ${CSR_NAME} -o jsonpath='{.status.certificate}')
+   ```shell-session
+   $ serverCert=$(kubectl get csr ${CSR_NAME} -o jsonpath='{.status.certificate}')
    ```
 
    -> If this process is automated, you may need to wait to ensure the certificate has been created.
    If it hasn't, this will return an empty string.
 
 2. Write the certificate out to a file.
 
-   ```bash
-   echo "${serverCert}" | openssl base64 -d -A -out ${TMPDIR}/vault.crt
+   ```shell-session
+   $ echo "${serverCert}" | openssl base64 -d -A -out ${TMPDIR}/vault.crt
    ```
 
 3. Retrieve Kubernetes CA.
 
-   ```bash
-   kubectl config view --raw --minify --flatten -o jsonpath='{.clusters[].cluster.certificate-authority-data}' | base64 -d > ${TMPDIR}/vault.ca
+   ```shell-session
+   $ kubectl config view --raw --minify --flatten -o jsonpath='{.clusters[].cluster.certificate-authority-data}' | base64 -d > ${TMPDIR}/vault.ca
    ```
 
 4. Create the namespace.
 
-    ```bash
-    kubectl create namespace ${NAMESPACE}
+    ```shell-session
+    $ kubectl create namespace ${NAMESPACE}
+    namespace/vault-namespace created
     ```
 
 5. Store the key, cert, and Kubernetes CA into Kubernetes secrets.
 
-   ```bash
-   kubectl create secret generic ${SECRET_NAME} \
-           --namespace ${NAMESPACE} \
-           --from-file=vault.key=${TMPDIR}/vault.key \
-           --from-file=vault.crt=${TMPDIR}/vault.crt \
-           --from-file=vault.ca=${TMPDIR}/vault.ca
+   ```shell-session
+   $ kubectl create secret generic ${SECRET_NAME} \
+       --namespace ${NAMESPACE} \
+       --from-file=vault.key=${TMPDIR}/vault.key \
+       --from-file=vault.crt=${TMPDIR}/vault.crt \
+       --from-file=vault.ca=${TMPDIR}/vault.ca
+
+   # secret/vault-server-tls created
```

Comment:
```suggestion
   secret/vault-server-tls created
```

Reply:
So, I did this on purpose because copy-paste of multi-line `shell-session` blocks will *also* include the output. So the `#` isn't there, but I didn't want people to accidentally try to execute the output. :)

Reply:
Ah, gotcha, TIL about the multi-line difference 👍